libsoftwarecontainer/src/gateway/networkgateway.cpp

```diff
@@ -102,25 +103,26 @@ ReturnCode NetworkGateway::readConfigElement(const json_t *element)
 
     // --- TEMPORARY WORKAROUND ---
     // in the wait of activate() being rewritten
-    if (e.defaultTarget == Target::ACCEPT) {
+    if ("ACCEPT" == e.m_defaultTarget) {
```

Comment:
But we only need to convert them to string in the end when calling iptables. I think that could be worth the effort.